tests/worker/workflow_sandbox/test_runner.py

```diff
@@ -263,10 +261,6 @@ async def test_workflow_sandbox_restrictions(client: Client):
 class DateOperatorWorkflow:
     @workflow.run
     async def run(self) -> int:
-        assert (
-            type(date(2010, 1, 20))
-            == temporalio.worker.workflow_sandbox._restrictions._RestrictedProxy
-        )
```

Comment:
Can you help me understand why this assertion no longer holds?

Reply:
It's because we no longer proxy instances of `date`, just classes and methods. You [agreed](https://github.com/temporalio/sdk-python/pull/757#discussion_r1941447284) with that course of action:

> I think we never meant for the instances to be restricted just the static methods

Shall we double-check this decision? Why were we proxing them originally -- just to prevent things like `my_datetime.now()`? (which would be unusual Python that I can't think of a use case for, but is technically a sandbox hole that we're opening up.)

Reply:
I remember now, thanks! So `type(type(date))` I guess is a `_RestrictedProxy` heh